src/components/formatters/TokenFormat/index.tsx

```diff
@@ -10,16 +10,25 @@ interface Props {
   token: string;
   style?: StyleProp<TextStyle>;
   decimalScale?: number;
+  fixedDecimalScale?: boolean;
 }
 
-function TokenFormat({ value, token, style, decimalScale }: Props) {
+function TokenFormat({
+  value,
+  token,
+  style,
+  decimalScale,
+  fixedDecimalScale = true,
```

Comment:
why is this true as default? in the real component is undefined as default

Reply:
My mistake